scripts/test_docker_image.sh

```
@@ -152,12 +201,10 @@
     # Create a unique container name for this test
     CONTAINER_NAME="wanderer-test-$(date +%s)"
     
-    # Start the container in the background
+    # Start the container in the background with all required environment variables
     docker run --name "$CONTAINER_NAME" -d -p 4000:4000 \
-      -e DISCORD_BOT_TOKEN="$DISCORD_TOKEN" \
-      -e CONFIG_PATH=/app/etc/wanderer_notifier.exs \
-      -e WANDERER_ENV=test \
-      -e WANDERER_FEATURE_DISABLE_WEBSOCKET=true \
+      -e DISCORD_BOT_TOKEN="$DISCORD_TOKEN" -e MAP_URL="http://somemap.com" -e MAP_TOKEN="some token" \
+      $EXTRA_ENV_VARS \
```

Shellcheck (0.10.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)